Makefile

```diff
@@ -39,28 +41,33 @@ static-doc:
 # but also ensures targets like benchmarks remain buildable.
 # NOTE: excludes must match test-workspace nextest params,
 #       otherwise some rebuilding will occur due to resolver = "2"
-build-workspace:
+build-workspace-wasmer_sys:
+	$(F) cargo build \
+		--workspace \
+		--locked \
+		--no-default-features \
+		--features $(DEFAULT_FEATURES),wasmer_sys --all-targets
+
+build-workspace-wasmer_wamr:
 	$(F) cargo build \
 		--workspace \
 		--locked \
-		--exclude hdk_derive \
-		--features $(A) --all-targets
+		--no-default-features \
+		--features $(DEFAULT_FEATURES),wasmer_wamr --all-targets
 
 # execute tests on all creates
-test-workspace:
+test-workspace-wasmer_sys:
 	cargo install cargo-nextest
 	$(F) RUST_BACKTRACE=1 cargo nextest run \
 		--workspace \
-		--exclude hdk_derive \
 		--locked \
-		--features $(A)
-	# hdk_derive cannot currently be tested via nextest
-	# https://github.com/nextest-rs/nextest/issues/267
-	$(F) cargo build \
-		--manifest-path crates/hdk_derive/Cargo.toml \
-		--locked \
-		--all-features --all-targets
-	$(F) RUST_BACKTRACE=1 cargo test \
-		--manifest-path crates/hdk_derive/Cargo.toml \
+		--no-default-features \
+		--features $(DEFAULT_FEATURES),wasmer_sys
```

Comment:
You've removed the `--all-targets` that was on the build step. Cargo recalculates the dependencies, and if the flags don't exactly match then it re-compiles the test binaries before running the tests, adding time to the CI run.

Reply:
good catch - reverted